crates/communication/src/client/parameter_subscription_manager.rs

```diff
@@ -159,6 +164,11 @@ pub async fn parameter_subscription_manager(
                         error!("{error}");
                     }
                 }
+                for sender in &notification_senders {
```

Comment:
Maybe also extract this code to a function?

Reply:
Döne